Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.py

```diff
@@ -584,46 +929,279 @@ def domain_pivot_command():
+        results.extend(response['results'])
+
+    output = []
     domain_context_list = []
+    risk_list = []
+    age_list = []
+    count = 0
     human_readable = 'No results found.'
-    outputs = {}  # type: Dict[Any, Any]
+    include_context = demisto.args().get('include_context', False)
 
     if response.get('results_count'):
-        for domain_result in response.get('results'):
-            human_readable_data.append(domain_result.get('domain'))
-            domain_context = create_domain_context_outputs(domain_result)
-            domain_context_list.append(domain_context.get('domaintools'))
+        for domain_result in results:
+            risk_score = domain_result.get('domain_risk', {}).get('risk_score')
+            first_seen = domain_result.get('first_seen', {}).get('value') if domain_result.get('first_seen') else False
+            output.append({'domain': domain_result.get('domain'), 'risk_score': risk_score})
+
+            if risk_score is not None:
+                risk_list.append(risk_score)
+            if first_seen:
+                timestamp = datetime.strptime(first_seen, "%Y-%m-%dT%H:%M:%SZ")
+                age = (current_date - timestamp).days
+                age_list.append(age)
+            if should_include_context(include_context):
+                domain_context = create_domain_context_outputs(domain_result)
+                domain_context_list.append(domain_context.get('domaintools'))
+
+            count += 1
+
+        average_risk = round(statistics.mean(risk_list), 2) if risk_list else "Unknown"
+        average_age = round(statistics.mean(age_list), 2) if age_list else "Unknown"
+
         outputs = {'DomainTools.PivotedDomains(val.Name == obj.Name)': domain_context_list}
         prune_context_data(outputs)
-        headers = ['Domains']
-        human_readable = tableToMarkdown('Domains for {}: {}.'.format(search_type, search_value),
-                                         human_readable_data,
+        headers = ['domain', 'risk_score']
+
+        sorted_output = sorted(output, key=lambda x: x['risk_score'] if x['risk_score'] is not None else -1, reverse=True)
+        human_readable = tableToMarkdown(f'Domains for {search_type}: {search_value} ({count} results, {average_risk} average risk, {average_age} average age)',
+                                         sorted_output,
                                          headers=headers)
 
-    return_outputs(human_readable, outputs, response)
+    results = CommandResults(
+        outputs_prefix='DomainTools.PivotedDomains',
+        outputs_key_field='Name',
+        outputs=domain_context_list,
+        readable_output=human_readable,
+        ignore_auto_extract=True
+    )
+    return_results(results)
+
+def to_camel_case(value):
+    str = f' {value.strip()}'
+    str = re.sub(r' ([a-z,A-Z])', lambda g: g.group(1).upper(), str)
+    return str
 
 
+def whois_history_command():
+    domain = demisto.args().get('domain')
```

Comment:
As domain here is mandatory It should be changed

```suggestion
    domain = demisto.args()['domain']
```

Reply:
updated